lms/static/scripts/frontend_apps/config.ts

```diff
@@ -205,28 +205,40 @@ export type Product = {
  * the backend code.
  */
 export type ConfigObject = {
+  // Configuration present in all modes.
   mode: AppMode;
   api: {
     authToken: string;
-    sync: APICallInfo;
-    viaUrl: APICallInfo;
+
+    // Only present in "basic-lti-launch" mode.
+    sync?: APICallInfo;
+    viaUrl?: APICallInfo;
   };
+  dev: boolean;
+  debug?: DebugInfo;
+  product: Product;
```

Comment:
"product" is not actually set in all modes yet. The intent here needs to be clarified.

Reply:
@marcospri is the intent that the "product" field would be used in multiple frontend apps, or only one specific one?